be/src/olap/olap_cond.cpp

```diff
@@ -262,28 +242,17 @@ bool Cond::eval(const std::pair<WrapperField*, WrapperField*>& statistic) const
         return operand_field->cmp(statistic.second) <= 0;
     }
     case OP_IN: {
-        FieldSet::const_iterator it = operand_set.begin();
-        for (; it != operand_set.end(); ++it) {
-            if ((*it)->cmp(statistic.first) >= 0 
-                    && (*it)->cmp(statistic.second) <= 0) {
-                return true;
-            }
-        }
-        break;
+        return (min_value_field->cmp(statistic.first) >= 0 && min_value_field->cmp(statistic.second) <= 0) ||
```

Comment:
Should this be:
`min_value_field->cmp(stat.second) <= 0 && max_value_field->cmp(stat.first) >= 0`
?